x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -360,14 +362,14 @@ FROM airports
 | LIMIT 3
 ;
 
-abbrev:keyword | city:keyword |       region:text | "COUNT(*)":long 
```

Comment:
Unrelated to this PR, but to previous [work](https://github.com/elastic/elasticsearch/pull/123589).